content/community/news/science-updates/2022-08-29-setiadi-blog-post.md

```diff
@@ -0,0 +1,180 @@
+
+![setiadi_2-2022](setiadi_2-2022.png "setiadi_2-2022")
+
+where <i>N</i> is the number of types of properties (density, solvation free energy, HG binding free energy etc.), <i>M<sub>n</sub></i> is the total number of properties of type <i>n</i>, <i>Y<sub>m</sub><sup>ref</sup></i> and <i>Y<sub>m</sub>(Θ)</i> are the experimental and calculated values of property <i>m</i>, respectively. The <i>weight<sub>n</sub></i> value gives weight to different property types and <i>denominator<sub>n</sub></i> scales the difference between the experimental and estimated properties. In this calculation, we set both of these values to 1 and 1 kcal/mol, respectively. Below is the plot of the objective function vs the optimization iteration. We see that the objective function progressively decreases with each iteration and reaches near zero at iteration 3.
+
+![setiadi_3-2022](setiadi_3-2022.png "setiadi_3-2022")
+
+In addition to the objective function, we can plot the binding free energy from each iteration to see how well the optimization performed. We see that the binding affinity is greatly overestimated (about 4 kcal/mol) compared to the experiment with the original parameters (red dash line). ForceBalance improves the binding affinity with each iteration and reaches a very good value after three iterations of optimization. 
+
+![setiadi_4-2022](setiadi_4-2022.png "setiadi_4-2022")
+
+In the figure below, I plot the evolution of the oxygen GB radius. ForceBalance shrinks the GB radius with each iteration to improve the binding free energy. The optimized radius is 1.14 Angstrom, down from the initial 1.50 Angstrom.  
+
+![setiadi_5-2022](setiadi_5-2022.png "setiadi_5-2022")
+
+Finally, to assess the quality of the optimized parameter, I have run some [benchmark calculations with OpenFF-Evaluator without ForceBalance](https://github.com/jeff231li/HG-GBSA-Optimization/blob/main/optimization/optimize.in). Below are the results for βCD with cyclobutanol, cis-4-methyl cyclohexanol, and benzoate acid.
+
+![setiadi_6-2022](setiadi_6-2022.png "setiadi_6-2022")
+ 
+We see that reducing the oxygen GB radius significantly improves the binding free energy calculation. The root-mean-squared error (RMSE) decreases from 2.92 kcal/mol to 1.00 kcal/mol.
+
+<b>Final Remarks</b>
+
+In this blog, I have shown an example of optimizing GB radii to fit host-guest binding data. The results here show some promise, but a larger training set is needed for production. There are also several things to consider:
+Which parameters should be optimized (i.e. how sensitive are different GB parameters to binding affinity)?
+How transferable are the optimized FF parameters to other host-guest complexes?
+What happens to other physical properties when we only fit the FF parameters to HG binding?
+How well do host-guest fitted FF parameters perform in protein-ligand systems?
```

Comment:
I think these lines are missing bullet points

Reply:
Could you check again?

Reply:
Looks great!